source/adapters/level_zero/v2/common.hpp

```diff
@@ -90,19 +96,19 @@ struct ze_handle_wrapper {
 };
 
 using ze_kernel_handle_t =
-    ze_handle_wrapper<::ze_kernel_handle_t, zeKernelDestroy>;
+    ze_handle_wrapper<::ze_kernel_handle_t, zeKernelDestroy, 0>;
```

Comment:
yeah, no, this is too awkward to use ;) I'd rather we create some sort of macro (or some clever lambda as parametr, dunno) that wraps all the destroy functions and calls ZE_CALL_NOCHECK inside of the wrapper.